python/loopsim.py

```diff
@@ -158,13 +190,18 @@ def _run(self, simSteps):
                 car["lane"] = traci.vehicle.getLaneIndex(v)
                 car["x"] = self._getX(car["edge"], position)
                 car["v"] = traci.vehicle.getSpeed(v)
+                car["maxv"] = traci.vehicle.getMaxSpeed(v)
+                car["f"] = traci.vehicle.getSpeedFactor(v)
```

Comment:
This doesn't change throughout the simulation, so can we retrieve and store this once?